docs/01-app/01-getting-started/09-fetching-data.mdx

```diff
@@ -53,6 +53,8 @@ export default async function Page() {
 }
 ```
 
+> **Good to know:** `fetch` responses are not cached by default. However, Next.js will [prerender](/docs/app/getting-started/partial-prerendering#static-rendering) the route and the output will be cached for improved performance. If you'd like to opt into [dynamic rendering](/docs/app/getting-started/partial-prerendering#dynamic-rendering), use the `{ cache: 'no-store' }` option. See the [`fetch` API Reference](/docs/app/api-reference/functions/fetch).
+
 #### With an ORM or database
```

Comment:
Should we mention that one can activate, https://nextjs.org/docs/app/api-reference/config/next-config-js/logging, to see if a fetch call is skips or hits cache?

Reply:
Good suggestion 👍🏼